video/out/vo_gpu_next.c

```diff
@@ -2058,8 +2058,14 @@ static void update_lut(struct priv *p, struct user_lut *lut)
     // Load LUT file
     char *fname = mp_get_user_path(NULL, p->global, lut->path);
     MP_VERBOSE(p, "Loading custom LUT '%s'\n", fname);
-    struct bstr lutdata = stream_read_file(fname, p, p->global, 100000000); // 100 MB
-    lut->lut = pl_lut_parse_cube(p->pllog, lutdata.start, lutdata.len);
+    const int lut_max_size = 600000000; // 600 MB
+    struct bstr lutdata = stream_read_file(fname, p, p->global, lut_max_size);
```

Comment:
Since you are touching this line, we might remove talloc parent here, it is freed after reading anyway.